A3A/addons/core/IntelMessages.hpp

```diff
@@ -0,0 +1,77 @@
+class IntelMessages {
+    notFound[] = {
+        "a wet noodle in the officers breast pocket.",
```

Comment:
Maybe a huge pain and the juice not worth the squeeze, but ideally we'd make this more easily extensible to other languages, whether they translate as is or add their own language-specific strings. Only major problem I foresee is reviewing could be difficult having to back-translate everything so we ensure people aren't surreptitiously adding abrasive/vulgar/etc language.

```suggestion
class IntelMessages {
    notFound[] = {
        __EVAL(localize "STR_intel_search_failure_1"),
        __EVAL(localize "STR_intel_search_failure_2"),
        ...
    };
};
```

Reply:
I was thinking: _who in their right mind is going to voluntarily translate 75 strings into another language._

But yeah, I kinda knew someone would suggest stringable.xml instead.

Compromize: until we finally agree to a list of required languages in dev-chat we leave it as is and I'll give it a go once we have a decision?

Reply:
We get Russian, Chinese, and Korean translations fairly regularly. The rest, eh. Doesn't really matter to me one way or the other, but would make it easier for others to do in future if they wanted, if all the strings are in the same place (or at least, the same *type* of files – the stringtables).

Reply:
No compromize, then, huh.